src/ExtensionsSample/Samples/TwilioSamples.cs

```diff
@@ -0,0 +1,102 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.WebJobs;
+using Newtonsoft.Json.Linq;
+using Twilio.Rest.Api.V2010.Account;
+using Twilio.Types;
+
+namespace ExtensionsSample.Samples
+{
+    // To use the TwilioSamples:
+    // 1. Configure your Twilio Account SID via the 'AzureWebJobsTwilioAccountSid' App Setting or Environment variable
+    // 2. Configure your Twilio Auth Token via the 'AzureWebJobsTwilioAuthToken' App Setting or Environment variable
+    // 3. Add typeof(TwilioSamples) to the SamplesTypeLocator in Program.cs
+    public static class TwilioSamples
+    {
+        /// <summary>
+        /// Demonstrates declaratively SMS message properties with parameter binding
+        /// to message properties.
+        /// </summary>
+        public static void ProcessOrder_Declarative(
+            [QueueTrigger(@"samples-orders")] Order order,
+            [TwilioSms(
+                From = "{StorePhoneNumber}",
+                Body = "{CustomerName}, we've received your order ({OrderId}) and have begun processing it!")]
+            out CreateMessageOptions messageOptions)
```

Comment:
Main change is that we now use the mutable CreateMessageOptions, whereas it was SMSMessage before. They overhauled their message APIs. The replacement for SMSMessage is MessageResource, and it is no longer mutable.